src/assets/styles/style.css

```diff
@@ -1,6 +1,6 @@
 @import url("./utils/variables.css");
```

Comment:
This file should be deleted :balloon:

Reply:
Yes, but it sure won't help if I delte it since I will get it back immediately...